Sources/Common/Core/Math/index.js

```diff
@@ -1883,6 +1920,18 @@ export function clampValue(value, minValue, maxValue) {
   return value;
 }
 
+export function clampVector(out, vector, minVector, maxVector) {
+  out[0] = clampValue(vector[0], minVector[0], maxVector[0]);
+  out[1] = clampValue(vector[1], minVector[1], maxVector[1]);
+  out[2] = clampValue(vector[2], minVector[2], maxVector[2]);
+}
+
+export function roundVector(out, vector) {
+  out[0] = Math.round(vector[0]);
+  out[1] = Math.round(vector[1]);
+  out[2] = Math.round(vector[2]);
+}
```

Comment:
I think those 2 methods should return something (i.e. `out`)

Maybe `out` should be an optional parameter:. e.g.
```export function roundVector(vector, roundedVector=[])```